cli/cmd/service/delete.go

```diff
@@ -17,6 +19,20 @@ func DeleteCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
+			if projectName != "" {
+				// If projectName is provided, delete the service from the specified project
+				_, err = client.RemoveProjectMemberService(cmd.Context(), &adminv1.RemoveProjectMemberServiceRequest{
```

Comment:
It feels a little weird that the `rill service delete` does removal when `--project` is passed. Also, is it correct that there's currently no way to remove a service's org role without deleting it?

For consistency with `rill user remove`, what do you think about adding a `rill service remove` command for removing service roles from orgs and projects, and keeping `rill service delete` only for deleting services?